assets/js/script.js

```diff
@@ -0,0 +1,190 @@
+$(function () {
+
+  console.log('Dom is ready')
+
+  var $container = $('.container')
+  var $topContainer = $('.topContainer')
+  var $mob = $('.mob')
+  var $player = $('.player')
+  var $bullet = $('.projectile')
+  var $body = $('body')
+  var $mobHP = $("#mobHP")
+  var $playerHP = $("#healthPoints")
+  var $hpBar = $(".hpBar")
+  var $startButton = $("#restart")
+  var fireDirection = ""
+  var keys = {37: false, 32: false, 39: false}
+  var playerHealth = 100
+  var mobHealth = 50
+  var currentMobsOnScreen = 0
+  var gameEnd = false
+  $bullet.hide()
+  $mob.css("left","205px")
+
+
+  // setInterval(checkHeight,300)
+
+  setInterval(function(){
+    if (gameEnd) { $(".landingScreen").show()}
+  },1000)
+
+  $startButton.on("click",restartGame)
+
+
+
+  function restartGame(){
+    $(".landingScreen").hide()
+    setInterval(startMob, 1000)
+    setInterval(bulletCollisionCheck, 10)
```

Comment:
Why is it set at 10? 1s = 1000, in a way 1000/10 =100 frame per seconds. Suggest to change to 25 frame per second (40) If is more than certain number of frame, human can not detect the changes anyway. Having more frame per second may affect the visual.